src/app/modules/main/profile_section/privacy/view.nim

```diff
@@ -66,8 +66,8 @@ QtObject:
   proc setUrlUnfurlingMode(self: View, value: int) {.slot.} =
     if self.getUrlUnfurlingMode() == value:
       return
-    self.delegate.setUrlUnfurlingMode(value)
-    self.urlUnfurlingModeChanged()
+    if self.delegate.setUrlUnfurlingMode(value):
+      self.urlUnfurlingModeChanged()
```

Comment:
It seems that we have a double-signalling here now.
Previous line will lead to this call-stack, calling same signal at the end.

1. `setUrlUnfurlingMode` 
2. `settings_service.saveUrlUnfurlingMode` 
3. `emit(SIGNAL_URL_UNFURLING_MODE_UPDATED)` 
4. `onUrlUnfurlingModeUpdated()`
5. `urlUnfurlingModeChanged()`

Reply:
So we probably don't need to emit `urlUnfurlingModeChanged` here at all. 
And then `setUrlUnfurlingMode` won't need to return `bool` result.

Reply:
Good catch! Thanks!
Fixed